app/interactors/create_group.rb

```diff
@@ -1,2 +1,13 @@
 class CreateGroup < StandardInteraction
+  def validate_input
+    context.fail!(errors: "invalid input") unless context.group_params
```

Comment:
I really love how you are working on improving this. I think specifying the required input as parameters would work great. You might need to figure out how to generalize that though, so that it can be applied to all interactions. Or maybe is just something that every interaction defines on its own.